<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ramWindow</class>
 <widget class="QWidget" name="ramWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cmdSet">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRPT7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DQS0 (Rainbow Dash side)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DQS7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdRead">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
